index.html

```diff
@@ -577,30 +598,92 @@ <h2>
         mode</a>.
       </p>
       <p>
-        Each <a>display mode</a>, except <code>browser</code>, has a
-        <dfn>fallback display mode</dfn>, which is the <a>display mode</a> that
-        the user agent can try to use if it doesn't support a particular
-        <a>display mode</a>. If the user agent does support a <a>fallback
-        display mode</a>, then it checks to see if it can use that <a>display
-        mode</a>'s <a>fallback display mode</a>. This creates a fallback chain,
-        with the <a>default display mode</a> (<code>browser</code>) being the
-        last item in the chain.
+        The fallback chain for:
+        <ol>
+          <li>
+            {{browser}} is «{{browser}}».
+          </li>
+          <li>
+            <a data-link-for="DisplayModeType">minimal-ui</a> is «
+            <a data-link-for="DisplayModeType">minimal-ui</a>, {{browser}}».
+          </li>
+          <li>
+            {{standalone}} is «{{standalone}},
+            <a data-link-for="DisplayModeType">minimal-ui</a>, {{browser}}».
+          </li>
+          <li>
+            {{fullscreen}} is «{{fullscreen}}, {{standalone}},
+            <a data-link-for="DisplayModeType">minimal-ui</a>, {{browser}}».
+          </li>
+        </ol>
+      </p>
+      <p>
+        For more advanced usages, the {{WebAppManifest/display_override}}
+        member can be used to specify a custom fallback order.
+      </p>
+      <p>
+        The <dfn>steps for determing the web app's chosen display mode</dfn> is
+        given by the following algorithm. The algorithm takes a
+        [=processed manifest=] |manifest:processed manifest| and returns a
+        [=display mode=].
+        <ol>
+          <li>
+            [=list/For each=] |candidate_display_mode:DisplayModeType| of
+            |manifest|.{{WebAppManifest/display_override}}:
+            <ol>
+              <li>
+                If the user agent supports the |candidate_display_mode|, then
+                return |candidate_display_mode|.
+              </li>
+            </ol>
+          </li>
+          <li>
+            [=list/For each=] |fallback_mode:DisplayModeType| of the
+            fallback_chain of |manifest|.{{WebAppManifest/display}}:
```

Comment:
Remove the underscore in "`fallback_chain`" since this is a prose term defined above without an underscore.

And link to the above definition (see above comment to use `<dfn>`). I think you can use `{{fallback chain}}` for this. Or maybe `[=fallback chain=]`. I usually guess until it works :)